Orange/widgets/visualize/owboxplot.py

```diff
@@ -378,6 +364,26 @@ def compute_score(attr):
             self.reset_attrs(domain)
         self.attribute = attribute
 
+    def _chi_square(self, group_var, attr):
+        # Chi-square with the given distribution into groups
+        # (see degrees of freedom in computation of the p-value)
+        if not attr.values or not group_var.values:
+            return 0, 2, 0
+        observed = np.array(
+            contingency.get_contingency(self.dataset, group_var, attr))
+        observed = observed[observed.sum(axis=1) != 0, :]
+        observed = observed[:, observed.sum(axis=0) != 0]
+        if min(observed.shape) < 2:
+            return 0, 2, 0
+        expected = \
+            np.outer(observed.sum(axis=1), observed.sum(axis=0)) / \
+            np.sum(observed)
+        dof = (len(group_var.values) - 1) * (len(attr.values) - 1)
+        return chisquare(
+            observed.ravel(), f_exp=expected.ravel(),
+            ddof=sum(observed.shape) - 2
+        ) + (dof, )
```

Comment:
We could replace the computation of expected values and the return by `return scipy.stats.chi2_contingency(observed)[:3]`, which does the same thing (https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.chi2_contingency.html).